cinema/tests/test_movie_session_api.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+
+from django.test import TestCase
+
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from cinema.models import Movie, Genre, Actor, MovieSession, CinemaHall
+
+
+class MovieSessionApiTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        drama = Genre.objects.create(
+            name="Drama",
+        )
+        comedy = Genre.objects.create(
+            name="Comedy",
+        )
+        actress = Actor.objects.create(first_name="Kate", last_name="Winslet")
+        movie = Movie.objects.create(
+            title="Titanic",
+            description="Titanic description",
+        )
+        movie.genres.add(drama)
+        movie.genres.add(comedy)
+        movie.actors.add(actress)
+        cinema_hall = CinemaHall.objects.create(
+            name="White",
+            rows=10,
+            seats_in_row=14,
+        )
+        MovieSession.objects.create(
+            movie=movie, cinema_hall=cinema_hall, show_time=datetime.datetime.now()
+        )
```

Comment:
You'd better always use `self.movie_session = ...`.
This will help to access needed values.